bundler/spec/spec_helper.rb

```diff
@@ -10,15 +10,30 @@ def require_common_spec(path)
 
 require "#{common_dir}/spec/spec_helper.rb"
 
-def bundler_2_available?
-  ENV["SUITE_NAME"] == "bundler2"
+module PackageManagerHelper
```

Comment:
This is an opinionated change, I tend to depart from the house style on preferring not to have test helper method defined globally.

That said, now there are a few helper methods related/derived from checking `ENV["SUITE_NAME"]`, I thought having a wrapper made much more sense.

Reply:
I also prefer this, thoughts on adding them to the rspec context though?

```ruby
RSpec.configure do |c|
  c.include PackageManagerHelper
end
```